### DISCUSSION TITLE
AWS_WEB_IDENTITY_TOKEN_FILE / IRSA support?

### DISCUSSION THREAD

**Discussion by tooptoop4** [juicedata/juicefs#1933] [General]
Instead of direct IAM role is IRSA supported for S3 auth? https://aws.amazon.com/blogs/opensource/introducing-fine-grained-iam-roles-service-accounts/

**Comment by zwwhdls** [comment]
Hi @tooptoop4 , 

You can set ServiceAccount in CSI Node pod, and mount pod will inherit it.

Refer to https://github.com/juicedata/juicefs-csi-driver/issues/167
